packages/cli/src/external-hooks.ts

```diff
@@ -70,6 +70,8 @@ type ExternalHooksMap = {
 	'workflow.afterUpdate': [updatedWorkflow: IWorkflowBase];
 	'workflow.delete': [workflowId: string];
 	'workflow.afterDelete': [workflowId: string];
+	'workflow.afterArchive': [workflowId: string];
+	'workflow.afterUnarchive': [workflowId: string];
```

Comment:
Have users asked for this? Reluctant to keep expanding external hooks unless there's a reason.

Reply:
No, this was just assumed on this feature.

If we don't want to create new hooks I can just drop them from this PR.

Reply:
Yes please, let's expand hooks only if needed.

Reply:
Mutasem brought up that the `afterDelete` hook is used here https://github.com/n8n-io/n8n-cloud/blob/6009a4bafab7946095de2f651210326a34873f90/packages/hooks-backend/src/usage.ts#L350C2-L350C13, and we would need to update the counts now also on `afterUnarchive` and `afterArchive`, so it looks like we need these after all? 🤔